tests/test_pdf2txt.py

```diff
@@ -126,3 +96,206 @@ def tests(fake_stdout):
         self.run_tests(
             tests, ['pdf2txt.py', 'samples/equations.pdf']
         )
+
+    @patch('tools.pdf2txt.create_file')
+    def test_create_html(self, mock_output):
+        # Test for the '.html' files
+        mock_output.return_value = True
+        laparams = LAParams()
+        debug = 0
+        # input option
+        password = b''
+        pagenos = set()
+        maxpages = 0
+        # output option
+        outfile = None  # If a file needs to be created
+        outtype = 'html'
+        imagewriter = None
+        rotation = 0
+        stripcontrol = False
+        layoutmode = 'normal'
+        encoding = 'utf-8'
+        scale = 1
+        caching = False
+        input_file = ['samples/simple1.pdf']
+
+        create_file_check = create_file(debug, caching, outfile, laparams,
+                                        imagewriter,
+                                        stripcontrol, scale, layoutmode,
+                                        pagenos,
+                                        maxpages, password, rotation, encoding,
+                                        input_file, outtype)
+        self.assertEqual(create_file_check, True)
+
+        outfile = 'html'
+        create_file_check = create_file(debug, caching, outfile, laparams,
+                                        imagewriter,
+                                        stripcontrol, scale, layoutmode,
+                                        pagenos,
+                                        maxpages, password, rotation, encoding,
+                                        input_file, outtype)
+        self.assertTrue(create_file_check)
+
+        self.assertTrue(main(['-t', 'html', 'samples/simple1.pdf']))
+
+    @patch('tools.pdf2txt.create_file')
+    def test_create_xml(self, mock_output):
+        # Test for the '.xml' files
+        mock_output.return_value = True
+        laparams = LAParams()
+        debug = 0
+        # input option
+        password = b''
+        pagenos = set()
+        maxpages = 0
+        # output option
+        outfile = None  # If a file needs to be created
+        outtype = 'xml'
+        imagewriter = None
+        rotation = 0
+        stripcontrol = False
+        layoutmode = 'normal'
+        encoding = 'utf-8'
+        scale = 1
+        caching = False
+        input_file = ['samples/simple1.pdf']
+
+        create_file_check = create_file(debug, caching, outfile, laparams,
+                                        imagewriter,
+                                        stripcontrol, scale, layoutmode,
+                                        pagenos,
+                                        maxpages, password, rotation, encoding,
+                                        input_file, outtype)
```

Comment:
When doing the change with the parameter i mentioned before, these functions can be changed to this:
```suggestion
        outtype = 'xml'
        input_file = ['samples/simple1.pdf']

        create_file_check = create_file(input_file, outtype)
```

Reply:
You are correct, this would make the tests easier to write but usually the functions are feed the parameter through the commandline which are parsed through. Moving the parameters in the function means that the argparser needs to redo too.

Reply:
I miss understood you earlier. I have changed now to default parameters, have a look at it when you have time.